### PR TITLE
Created skeleton for frontend and backend, installed mui, express, do…

### PR DESCRIPTION
**Pull request by MashaSavchuk**
…tenv, nodemon, react-router-dom

### CODE REVIEW

File: api/server.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("Hello synergy!!!!!")
```
Comment (OyakiMorse) [inline api/server.js]: such records are better not to add to the git, as they are not useful

File: client/src/components/App/App.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+
```
Comment (OyakiMorse) [inline client/src/components/App/App.js]: Empty spaces also will be remove or use Prettier, Eslint, etc for formatting code

File: client/src/setupTests.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```
Comment (OyakiMorse) [inline client/src/setupTests.js]: Comments also remove

File: api/.gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+/node_modules
+/package-lock.json
```
Comment (OyakiMorse) [inline api/.gitignore]: Need remove  package-lock.json from .gitignore

**Review by OyakiMorse — COMMENTED** [review]
Please read the comments below. Thanks

**Review by Yuliia-Sydorovych — APPROVED** [review]

**Review by AntonyJedi — APPROVED** [review]

**Review by OyakiMorse — APPROVED** [review]
cool

**Review by AntonyJedi — COMMENTED** [review]
Great

**Pull request merged**